packages/app-root/src/app/users/[login]/stats/UserStatsContainer.js

```diff
@@ -28,7 +28,18 @@ function UserStatsContainer({
   projectId,
   startDate
 }) {
-  const { adminMode, isLoading, user } = useContext(PanoptesAuthContext)
+  const { adminMode, isLoading, user: authUser } = useContext(PanoptesAuthContext)
+
+  // fetch user with created_at property
+  const {
+    data: user,
+    error: userError,
+    isLoading: userLoading
+  } = usePanoptesUser({
+    authUser,
+    login,
+    requiredUserProperty: 'created_at'
+  })
```

Comment:
This request is now duplicated here and in lib-user > UserStatsContainer. The response will be cached so it's not the end of the world, but just curious if you considered passing `user` all the way down to UserStats instead of duplicating the fetch for data. Is it because the dev App setup requires a fetch for `created_at` when it doesn't have app-root as its parent app?

Reply:
That's a great point, the user requests are being duplicated, and the group request similarly feels like there's a refactor here. Essentially how app-root is requesting the stats sources then those same stats sources are being requested in lib-user isn't ideal. I'm going to merge this as-is but will open an issue (or maybe just a quick draft PR) to better document and we can revisit.